README.md

```diff
@@ -12,6 +12,14 @@ Finally, the metrics are available in [grafana](https://grafana.com/) to build d
 and set it up as `SpaceAuditor` on each monitored space.
 - Terraform and the [cloudfoundry plugin](https://registry.terraform.io/providers/cloudfoundry-community/cloudfoundry/latest)
 
+## Redis Services
+If your application uses REDIS you may want to include a REDIS Metrics Exporter for each instance of REDIS you use. This is accomplised by passing in an array of strings. Each string takes the form
```

Comment:
accomplished

Reply:
I really do need to learn how to type ..